src/_resources/fact-sheet-ods-overview.yml

```diff
@@ -0,0 +1,8 @@
+---
+date: "2023-11-08T17:00:00-07:00"
+title: "Overview: Operational Data Standard"
+asset: https://drive.google.com/file/d/1U8DKyqj9Nx5J0rpgVlHMBsctEm0I6IIO/view
```

Comment:
The latest version of this asset was uploaded in this comment: https://github.com/cal-itp/calitp.org/issues/300#issuecomment-1932527083

Reply:
Updated to new link. The hand is correct now 😆 
<img width="628" alt="image" src="https://github.com/cal-itp/calitp.org/assets/3673236/043d8ccc-1080-4ab9-92ab-977d3bfc816b">